heksher/clients/async_client.py

```diff
@@ -241,6 +253,14 @@ async def get_settings(self) -> Dict:
         settings = SettingsOutput.parse_obj(response.json()).to_settings_data()
         return settings
 
+    def on_update_error(self, exc):
```

Comment:
1. Why aren't on_update_error and on_update_ok tested?
2. Wouldn't it be an easier usage if instead of inheriting from AsyncHeksherClient just to override these methods, you'd expose the callbacks set by a set_on_update_error_callback or similar, or even by passing it via the c'tor?

Reply:
1. they are tested, they're fully covered
2. I don't think so. This is meant to allow users to create adapters to whatever they need, these adapters will probably already extend the client classes for their own needs.